### DISCUSSION TITLE
Welcome to Prism Discussions!

### DISCUSSION THREAD

**Discussion by codenameakshay** [Hash-Studios/Prism#211] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jahnkit7** [comment]
Hello, Nice job! Congratulations to all of you.
Very nice application!
I would like to know, does the project work with a Backend?

**Reply by codenameakshay** [reply]
Yeah, it uses Google Firebase as the backend, with GitHub  as a storage space for all wallpapers/setups.
